schema/README.md

```diff
@@ -0,0 +1,7 @@
+## Hello! Welcome to Fleet's osquery tables documentation.
+
+This file contains additional documentation that we add on top of the existing documentation for osquery.
```

Comment:
Instead of:
> on top of the existing documentation for osquery.

Let's say why:

> on top of the existing documentation for osquery to make the documentation of each table more useful for Fleet users.